Sources/UIImageExtensions.swift

```diff
@@ -13,7 +13,7 @@ import UIKit
 extension UIImage {
     
     /// EZSE: Returns base64 string
-    var base64: String {
+    public var base64: String {
```

Comment:
Cool. I presume there are unit tests already written for this?

Reply:
It's actually the only test for UIImage: https://github.com/goktugyil/EZSwiftExtensions/blob/master/EZSwiftExtensionsTests/UIImageTests.swift#L16